.github/workflows/build_msi_installer.yml

```diff
@@ -108,6 +108,11 @@ jobs:
           setupType: promptflow_with_extra
           scriptPath: ${{ env.testWorkingDirectory }}
 
+      - name: Generate portable promptflow
+        run: | 
+          echo "Python_ROOT_DIR: $env:Python_ROOT_DIR"
+          Compress-Archive -Path $env:Python_ROOT_DIR -DestinationPath promptflow-${{ steps.get-version.outputs.version }}.zip
```

Comment:
Let's only zip the output of pyinstaller

Reply:
done